crates/paths/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "spacetimedb-paths"
+version.workspace = true
+edition.workspace = true
+rust-version.workspace = true
```

Comment:
minor thing - can you add a description and license file for this? Otherwise it will fail to publish. (I think our convention for other crates is to just use an empty license file called `LICENSE`).